create_derivatives/routines.py

```diff
@@ -86,16 +85,16 @@ def __init__(self):
     def run(self, replace):
         uploaded_bags = []
         for bag in Bag.objects.filter(process_status=Bag.MANIFESTS_CREATED):
-            pdf_dir = join(bag.bag_path, 'data', 'PDF')
-            jp2_dir = join(bag.bag_path, 'data', 'JP2')
-            manifest_dir = join(bag.bag_path, 'data', 'MANIFEST')
+            pdf_dir = Path(bag.bag_path).joinpath('data').joinpath('PDF')
+            jp2_dir = Path(bag.bag_path).joinpath('data').joinpath('JP2')
+            manifest_dir = Path(bag.bag_path).joinpath('data').joinpath('MANIFEST')
```

Comment:
See what I've done above.

You'll also likely need to call `str` on the `Path` object in order to get a string representation.

Reply:
Yeah, so I think [this](https://docs.python.org/3/library/pathlib.html#correspondence-to-tools-in-the-os-module) threw me off.